### PR TITLE
[feat]: [BT-1357]: Updating application images with UBI base

### PR DESCRIPTION
**Pull request by diptiman-adak**
## Checklist
- [ ] I've documented the changes in the PR description.
- [ ] I've tested this change either in PR or local environment.
- [ ] If hashcheck failed I followed [the checklist](https://harness.atlassian.net/wiki/spaces/DEL/pages/21016838831/PR+Codebasehash+Check+merge+checklist) and updated this PR description with my answers to make sure I'm not introducing any breaking changes.

## Comment Triggers
<details>
  <summary>Build triggers</summary>
  
- Feature build: `trigger feature-build`
- Immutable delegate `trigger publish-delegate`
</details>

<details>
  <summary>PR Check triggers</summary>
  
  You can run multiple PR check triggers by comma separating them in a single comment. e.g. `trigger ti0, ti1`
  
- Compile: `trigger compile`
- CodeFormat: `trigger codeformat`
- MessageMetadata: `trigger messagecheck`
- Recency: `trigger recency`
- BuildNumberMetadata: `trigger buildnum`
- runDockerizationCheck: `trigger dockerizationcheck`
- runAuthorCheck: `trigger authorcheck`
- Checkstyle: `trigger checkstyle`
- PMD: `trigger pmd`
- TI-bootstrap: `trigger ti0`
- TI-bootstrap1: `trigger ti1`
- TI-bootstrap2: `trigger ti2`
- TI-bootstrap3: `trigger ti3`
- TI-bootstrap4: `trigger ti4`
- FunctionalTest1: `trigger ft1`
- FunctionalTest2: `trigger ft2`
- CodeBaseHash: `trigger codebasehash`
</details>

---
This change is [Reviewable](https://reviewable.io/reviews/harness/harness-core/34075)

### CODE REVIEW

File: dockerization/application-runtime-images/cv-nextgen/Dockerfile-cv-nextgen-cie-jdk [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# to be used when building in Jenkins
```
Comment (shivagowda) [inline dockerization/application-runtime-images/cv-nextgen/Dockerfile-cv-nextgen-cie-jdk:1]: nit: Jenkins?

File: dockerization/application-runtime-images/analyser-service/Dockerfile-analyser-service-cie-jdk [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# to be used when building in Jenkins
```
Comment (shivagowda) [inline dockerization/application-runtime-images/analyser-service/Dockerfile-analyser-service-cie-jdk:1]: nit: Jenkins?

Comment (shivagowda) [inline dockerization/application-runtime-images/analyser-service/Dockerfile-analyser-service-cie-jdk:1]: pls add copyright header to all files, including the dockerfiles that were merged earlier

File: dockerization/application-runtime-images/analyser-service/scripts/run.sh [hunk 3]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+# Copyright 2021 Harness Inc. All rights reserved.
```
Comment (shivagowda) [inline dockerization/application-runtime-images/analyser-service/scripts/run.sh:2]: change to `2022`

**Review by shivagowda — APPROVED** [review]
> Reviewed 44 of 45 files at r2, 2 of 2 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/34075)* status: :shipit: complete! ✓

**Review by USB-05 — APPROVED** [review]
> Reviewed 14 of 45 files at r2, 17 of 18 files at r5, 2 of 2 files at r6, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/34075)* status: :shipit: complete! ✓

**Review by vikas-harness — APPROVED** [review]
> Reviewed 27 of 45 files at r2, 17 of 18 files at r5, 2 of 2 files at r6, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/34075)* status: :shipit: complete! ✓

**Pull request merged**